lib/run_printer.py

```diff
@@ -112,8 +112,12 @@ def get_run_checker():
 
 
 
-def _exit_success(run):
-    print(json.dumps(run, indent=2))
+def _exit_success(run, out_file):
+    if out_file is not None:
```

Comment:
What's the difference between `if out_file is not None:` and `if out_file:` ?

Reply:
No difference.
But i saw it being used [HERE](https://github.com/awslabs/aws-build-accumulator/blob/release/litani#L383-L385) and to be consistent I used the same.

Reply:
Updated